macros/shared

```diff
@@ -239,8 +241,21 @@ CROSS_COMPILATION_CONF_EOF\
 %_source_payload w1.zstdio
 %_binary_payload w1.zstdio
 
+# Invocation for the check-fips buildroot policy script.
+%__cross_check_fips_cmd %{shrink: \
```

Comment:
Shouldn't this be `__cross_check_fips_go_cmd`, since we may use it for checks in Rust binaries, or are you planning to use a different check for that?

Reply:
> Shouldn't this be __cross_check_fips_go_cmd, since we may use/declare a different check in Rust binaries, or are you planning to do something different for those programs?

I'd like to handle both Rust and Go in the same script if possible. If not, I can extend this macro to run both scripts.